source/logs/command-history.txt

```diff
@@ -32,3 +32,37 @@ text editor:
 
    * - Windows
      - ``%UserProfile%/.mongodb/mongosh/mongosh_repl_history``
+
+Starting in |mdb-shell| 2.4.0, you can use the ``history()`` command to
+view a list of previously executed commands. For example:
+
+.. code-block:: javascript
+
+   history()
+
+The following example output shows a list of commands in an array of
+strings:
+
+.. code-block:: javascript
+   :copyable: false
+
+   [
+     'db.salesFromFile.find()',
+     '   let salesCollection = JSON.parse( db.sales.find().toArray() )   ',
+     '   let salesCollection = JSON.parse( db.sales.find() )   ',
+     '   let salesCollection = EJSON.stringify( db.sales.find().toArray() )   ',
+     "   fs.writeFileSync( 'salesDeserialize.json', salesCollection ) ",
```

Comment:
Why the indentation? Not saying it's bad, just surprising

Reply:
Used different example